pxr/usdImaging/usdviewq/appController.py

```diff
@@ -2185,16 +2188,28 @@ def _findPrims(self, pattern, useRegex=True):
             pattern = pattern.lower()
             isMatch = lambda x: pattern in x.lower()
 
-        matches = [prim.GetPath() for prim
-                   in Usd.PrimRange.Stage(self._dataModel.stage,
-                                             self._displayPredicate)
-                   if isMatch(prim.GetName())]
+        if self._dataModel.viewSettings.showPrimDisplayNames:
+            matches = [prim.GetPath() for prim
+                    in Usd.PrimRange.Stage(self._dataModel.stage,
+                                                self._displayPredicate)
+                    if ((prim.HasAuthoredMetadata("displayName") and isMatch(prim.GetMetadata("displayName"))) or isMatch(prim.GetName()))]
```

Comment:
Discussed this with our designers, and they feel it may be too confusing to match against names you can't see, so can we only match against a non-empty displayName, when present?  And if that change doesn't bring us under 80 columns, can we do a continuation, also, please?

Reply:
Sorry, I think I'm misunderstanding your concern, @erslavin ?  When "Show Display Names" is on, then search should consult displayName, but **only** for prims on which it is authored, i.e. we should just match based on what you see in the browser.  I don't think that will be surprising, and I would still advocate for having the default be "on" because authoring displayNAme and _not_ seeing it reflected in the browser is surprising, I'd argue (acknowledging again that's already the boat we're in with the property browser).

Reply:
ok that was my misunderstanding, let me update again thanks!

Reply:
Ok hopefully we are in agreement now.